Source/Core/DolphinQt/Config/GraphicsModListWidget.cpp

```diff
@@ -3,6 +3,11 @@
 
 #include "DolphinQt/Config/GraphicsModListWidget.h"
 
+#ifdef _MSC_VER
+// Disable warning for hiding global declaration (qt uint vs fmt uint)
+#pragma warning(disable : 4459)
```

Comment:
Ah, I didn't see them because they weren't listed as patch files... I assume 357dc72f97d812bf5d393d5f7216af8aa2d2c95a and 7b8e6c5b3643bb3ec09de3be5598d5e35caa3986?

Reply:
Looks like 357dc72f97d812bf5d393d5f7216af8aa2d2c95a isn't needed anymore, but I was able to cherry-pick 7b8e6c5b3643bb3ec09de3be5598d5e35caa3986 and it seems to work.